packages/graphql/tests/schema/interfaces.test.ts

```diff
@@ -487,11 +487,11 @@ describe("Interfaces", () => {
               id_EQUAL: ID @deprecated(reason: \\"Aggregation filters that are not relying on an aggregating function will be deprecated.\\")
             }
 
-            interface MovieNode @something(something: \\"test\\") {
+            interface MovieNode {
```

Comment:
the directive is missing here, user defined directives are not propagated anymore